profiles-sdk-examples/main-app/src/main/resources/conf/driverTemplate.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
```

Comment:
This template should be built in to the base spark-template going forward. Do we need to explicitly copy it over in the main-apps Dockerfile ?

Reply:
Nvm, I think the parent folder gets copied over

Reply:
There is something more that needs to be done here around the base sdk image and examples images having some duplicated files like spark_submit.py. I am not sure where that will land but am just following what we have been doing for now.

Reply:
Makes sense, in my mind the script fits as part of the packaging of the SDK as skill so maybe it'll end up as part of a  "skill template" but we'll see.